keyboards/kc60se/readme.md

```diff
@@ -1,15 +1,63 @@
 # kc60se
 
-![kc60se](imgur.com image replace me!)
+Keyboard Maintainer: [Blake C. Lewis](https://github.com/BlakeCLewis), [MechMerlin](https://www.github.com/mechmerlin)       
 
-A short description of the keyboard/project
+Hardware Supported: KC60SE
+
+Hardware Availability: PCB is nolonger available
+
+## Description:
+  * 60% keyboard PCB;
+  * supports ANSI, ISO, and HHKB layouts.
 
-Keyboard Maintainer: [Blake C. Lewis](https://github.com/BlakeCLewis), [MechMerlin](https://www.github.com/mechmerlin)       
-Hardware Supported: KC60 SE
-Hardware Availability: [Aliexpress](https://www.aliexpress.com/store/product/Free-shipping-GH60-PCB-KC60-SE-Fully-Programmable-For-DIY-Mechanical-Keyboard-Poker-Faceu-HHKB-Support/429151_32799437588.html?spm=2114.12010608.0.0.2995e5c0hNRgMH)   
 
-Make example for this keyboard (after setting up your build environment):
+### Enter the Bootloader
+  * As delivered, BOOTMAGIC_ENABLE = yes:
+    * unplug keyboard;
+    * while holding down 'Space' 'B', plug in USB cable and continue to hold for 8 seconds;
+    * 'lsusb' should report 'Atmel Corp. atmega32u4 DFU bootloader' instead of 'feed 6060';
+    * in bootloader LEDs are off &amp; keyboard does not work.
+  * New firmware, BOOTMAGIC_ENABLE = lite:
+    * holding down esc while pluging in usb.
+  * Hardware way: 
+    * momentary jump pads 5 &amp; 6 to enter bootloader;
 
+### Compile the default keymap:
     make kc60se:default
 
-See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+### See [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
+
+### NKRO
+  * N or 6 Key Rollover NKRO/6KRO
+  * navigating bios of computer may require 6KRO
+  * default keymap has fn(T) mapped to toggle NKRO/6KRO
+  * test: press 8 keys at same time, if 6, then 6KRO, if 8, then NKRO
+
+### DIP switches enable/disable GUI keys, the second keys from the ends on the bottom row:
+  * K41/K4C are WIN/GUI and will disrupt games;
+  * this can be done by mapping a key to toggle these keys
```

Comment:
The `GUI_OFF` keycode will disable the GUI keys. `GUI_ON` keycode to enable them again.